lib/propolis/src/block/crucible.rs

```diff
@@ -154,6 +154,16 @@ impl CrucibleBackend {
         }))
     }
 
+    /// Return the block size of this Crucible backend, if it can be determined.
+    pub async fn block_size(&self) -> Option<u32> {
+        self.state
```

Comment:
I'm not sure what happens if we call this before the upstairs is active.
When you run this, do you get a block size back from the volume?
If it works, then you are calling it after activation or I don't know what I'm talking about.  Possibly both.

Reply:
Welp, I'm about to test it, so I'll let you know! We also get the block size in the API request from the sled-agent, so it's possible we can pass it down from there to where we need it in another way, rather than asking Crucible.